lua/rustaceanvim/neotest/init.lua

```diff
@@ -44,7 +44,23 @@ end
 ---@param file_path string
 ---@return boolean
 NeotestAdapter.is_test_file = function(file_path)
-  return vim.endswith(file_path, '.rs')
+  if file_path:find('target/%a+/build') or not vim.endswith(file_path, '.rs') then
```

Comment:
suggestion: I think [`neotest.Adapter.filter_dir`](https://github.com/nvim-neotest/neotest/blob/89a5b1fc22ee5d459262955fe564c3d14e35fd67/lua/neotest/adapters/interface.lua#L20) is probably better suited for this.

Reply:
Nice! I'll take a look at that.